pkg/api/configsync/v1beta1/resource_override.go

```diff
@@ -162,8 +162,8 @@ type ContainerLogLevelOverride struct {
 
 	// logLevel specifies the log level override value for the container.
 	// The default value for git-sync container is 5, while all other containers will default to 0.
```

Comment:
Is the default log level for otel-agent -1 or 0?

Reply:
The default is 0, which maps to `INFO`